### DISCUSSION TITLE
Cannot publish App in github marketplace

### DISCUSSION THREAD

**Discussion by MohdRashid01** [community/community#24697] [Apps, API and Webhooks]
<p>Hi,</p>
<p>i want to publish an app in github marketplace i try in both method from organistaion and from personal account both not showing draft a release i added action.yaml file in root here is my repo link</p>
<p>From Organization</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/apisec-inc/apisec-run-scan">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/apisec-inc/apisec-run-scan" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/7812871/181086296-2126200a-1836-4a52-979f-936ff4e31821.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c89935f5c6e2b99446de7a1f396bffe00d350fa_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c89935f5c6e2b99446de7a1f396bffe00d350fa_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/c/7c89935f5c6e2b99446de7a1f396bffe00d350fa.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c89935f5c6e2b99446de7a1f396bffe00d350fa_2_10x10.png"></div>

<h3><a href="https://github.com/apisec-inc/apisec-run-scan" target="_blank" rel="noopener nofollow ugc">GitHub - apisec-inc/apisec-run-scan: This action triggers on-demand scans for...</a></h3>

  <p>This action triggers on-demand scans for projects registered in APIsec. - GitHub - apisec-inc/apisec-run-scan: This action triggers on-demand scans for projects registered in APIsec.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>From Personal Account<br>
<a href="https://github.com/MohdRashid01/test-scanner" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/MohdRashid01/test-scanner</a></p>
<p>in Both i cannot publish an app and also in organization yesterday i was able to see publish a app and draft a release but today not showing</p>
<p>Help me…</p>

**Comment by jainamoswal** [comment]
<p>Upps, you did an typo (indentation error) at line <a href="https://github.com/MohdRashid01/test-scanner/blob/34dd966e52ac4018402d3558f7713a95df713729/action.yaml#L30" rel="noopener nofollow ugc">30.</a></p>
<p>Just remove one space from that line. And make sure they all are in same indent.</p>
<pre><code class="lang-auto">runs:
  using: "composite"
  steps:
    - run: | 
         wget https://raw.githubusercontent.com/intesar/FX-Scripts/master/fx_job_invoke_script.sh?token=AElo1orWSWln6w6y5uX0-1ylzPH8TX_3ks5bqJBywA%3D%3D -O fx_job_invoke_script.sh;bash fx_job_invoke_script.sh ${{inputs.apisec-username}} ${{ inputs.apisec-password }} ${{ inputs.apisec-project }} ${{ inputs.apisec-region }} ${{ inputs.apisec-environment }} ${{ inputs.apisec-scan-job }}
      shell: bash
</code></pre>
<p><strong>Hope my answer solves your issue.</strong><br>
<em>~ Jainam</em></p>

**Comment by MohdRashid01** [comment]
<p>Thanks for notice but after changing still i didnt get publish app into marketplace (draft a release) option .</p>

**Comment by jainamoswal** [comment]
<p>I can’t see any commit about you changed that typo.<br>
Anyways, I had forked and tested it. It worked and showed me that option and still showing. I don’t know why it’s not showing to you. :thinking:</p>
<p>I’ve listed some options below which could help you.</p>
<p>Maybe try again with</p>
<ul>
<li>Change that <a href="https://github.community/t/cannot-publish-app-in-github-marketplace/200073/2">typo</a>
</li>
<li>Adding <code>branding</code> meta data in that action.yml</li>
<li>Change file name with .yml from .yaml :thinking:
</li>
</ul>
<p>~ Jainam</p>

**Comment by MohdRashid01** [comment]
<p>Thanks it solve my issue<br>
by</p>
<p>changing yaml to yml</p>

**Comment by jainamoswal** [comment]
<p>If that helped, please mark my reply as “Answer”. :blush:</p>
<p>Thanks.</p>
